extra/2-piping.js

```diff
@@ -16,30 +16,31 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+return a + b;
 }
 
-function multiply() {
-
+function multiply( a, b) {
+return a * b;
 }
 
-function format() {
+function format(number) {
+  return `£${number}`;
 
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+const badCode = format((startingValue + 10)*2);
 
 /* BETTER PRACTICE */
 
-let goodCode =
-
-/* ======= TESTS - DO NOT MODIFY ===== 
-There are some Tests in this file that will help you work out if your code is working.
+let addition = add(startingValue, 10);
+let multification = multiply(addition, 2);
+let goodCode = format(multification);
```

Comment:
LOVE it. This is so tidy, clear and well labelled (watch out for the p in multification though).